src/hooks/Bonds.ts

```diff
@@ -1,13 +1,13 @@
-import { useSelector } from "react-redux";
 import { useEffect, useState } from "react";
 import orderBy from "lodash/orderBy";
+import { useAppSelector } from ".";
```

Comment:
this is a weird import.

Reply:
haha, i've changed it so it is more informative